NOTICE

```diff
@@ -40,3 +40,7 @@ Terms: https://github.com/MatthewYork/DateTools/blob/master/LICENSE
 - OCMapper 
 Url: https://github.com/aryaxt/OCMapper
 Terms: https://github.com/aryaxt/OCMapper/blob/master/License.txt
+
+- UIDevice-Hardware
+Url: https://github.com/erichoracek/UIDevice-Hardware
+Terms: https://github.com/erichoracek/UIDevice-Hardware/blob/master/LICENSE
```

Comment:
Are we sure we need to use this third party library?

It seems like all it does is map `getSysInfoByName:"hw.machine"];` to a slightly more user-friendly string: https://github.com/erichoracek/UIDevice-Hardware/blob/master/UIDevice-Hardware.m#L44.

Since the model identifier needs to be used by us, we can look it up ourselves and not rely on third party code.

Reply:
Well iPhone X is much more readable than iPhone 3,4
I can even copy this function inside the app and keep it updated.

Reply:
I think that since this info is developer information for us to investigate the bug, we can do the lookup ourselves.

If you still think it's useful to you to have it be resolved on the users device, then maybe vendorising (copy that bit of code into our codebase) just that function is the way to go as it's pretty small.
Keep in mind, though, that we will have to update this function every time a new phone comes out and it's probably easier to just use the model identifier and maintain a lookup table that is external to the apps.

I am thinking of this from the perspective of maybe integrating the bug reporting email alias inside of some automated bug reporting workflow.

Reply:
I'm still not sold on this. 
This library is always updated (at least until now) and even if it will not be we will get the iPhone 15,75 string. (and in that case we can clone it and mantain ourselves)
It's much more difficult for us to "translate" the iPhone version to the model name every time we get an email.

Reply:
Is there a reason why you need to have the brand name of the phone and the model ID is not enough?

My concern is that you may end up with two users (which have two different version of the app with two different version of this library) submitting a different model string, but actually are on the same phone.

For example say the `iPhone XI` comes out (`iPhone12,8`) and it takes us some time to ship an update to the library, we may have the same time people sending us `iPhone XI` & `iPhone 12,8`, which could be confusing.

In any case I don't think it's a super big deal, but if we can avoid including a third party library (which is more code that can include more bugs) it would be better imho.

Reply:
It is enough but You will have to go trought every model manually and "translate" it into human readable string. 
Even if we have double string `iPhone XI` and `iPhone12,8` for a short period of time I don't think is such a big deal.
As you are the master of automation, automatizing every single bit, I don't really know why you prefer this solution!

Reply:
@hellais your call 
I still prefer automating this little bit of translating model into names, but it's up to you.